math/lcm.go

```diff
@@ -0,0 +1,49 @@
+// Formula from https://en.wikipedia.org/wiki/Least_common_multiple
+// Find the least common multiple of two integers
+// by using the greatest common divisor.
+
+package main
+
+import (
+	"fmt"
+)
+
+// abs: absolute value for an integer
+func abs(a int) int {
+	if a < 0 {
+		return -a
+	}
+	return a
+}
+
+// gcd: greatest common divisor -
+// the largest positive integer that divides both a and b
+func gcd(a, b int) int {
```

Comment:
Remove this function and use the `Gcd` function that is already implemented in the repository.